### PR TITLE
update release note for SSDT for VS2017 15.9.6

### PR DESCRIPTION
**Pull request by MSraye**
update release note for SSDT for VS2017 15.9.6

### CODE REVIEW

**Comment by PRMerger18** [comment]
@MSraye : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
Hi @MSraye Release notes should be updated in the private repo so the revisions can be checked on staging and pass quality validations before merging. Would you move this commit to the private repo? Let me know if you need information about how to access the private repo; I can email it to you. Thanks. We'll close this PR.

**Comment by ktoliver** [comment]
Hi @MSraye I've reopened the PR so it can move forward with review. I'll send email with information about working in the private repo. Thanks.

**Comment by ktoliver** [comment]
@dzsquared Would you review the proposed update to your article? If you approve the PR, we can merge it. Thanks.

File: docs/ssdt/release-notes-ssdt.md [hunk 1]
```diff
@@ -40,6 +40,35 @@ GeneMi , 2019/03/22.
 P.S.  there's no need to keep this large HTML comment indefinitely.
 -->
 
+## 15.9.6,&nbsp; SSDT for VS 2017
+
+_Released:_ &nbsp; Oct 27, 2020
```
Comment (dzsquared) [inline docs/ssdt/release-notes-ssdt.md:45]: Let's fix this date to one not in the future

Comment (MSraye) [reply docs/ssdt/release-notes-ssdt.md:45]: Nice catch for the typo error. Will update it.

Comment (MSraye) [reply docs/ssdt/release-notes-ssdt.md:45]: Created new PR #5389 and will close this one.

**Pull request closed without merging**